.github/workflows/builder_node_slsa3.yml

```
@@ -268,7 +268,7 @@ jobs:
 
           # cp output into upper folder to make the tarball accessible to
           # next step.
-          echo '::set-output name=filename::$TARBALL'
+          echo 'filename=$TARBALL >> $GITHUB_OUTPUT'
```

I don't think this actually works (and was broken before this too). You need to double quote this or the environment variables are not expanded.
```suggestion
          echo "filename=$TARBALL >> $GITHUB_OUTPUT"
```